### PR TITLE
K8TSPG-409 Get Started section 

### PR DESCRIPTION
**Pull request by nastena1606**

### CODE REVIEW

File: docs/backups-storage.md [hunk 1]
```diff
@@ -0,0 +1,276 @@
...
+
+    1. Encode the Azure Storage credentials and the pgBackRest repo name that you will use for backups with base64. In this example, we are using `repo1`.   
+
+        === "Linux"    
+
+            ``` {.bash data-prompt="$" }
+            $ cat <<EOF | base64 --wrap=0
+            [global]
+            repo1-azure-account=<AZURE_STORAGE_ACCOUNT_NAME>
+            repo1-azure-key=<AZURE_STORAGE_ACCOUNT_KEY>
```
Comment (nmarukovich) [inline docs/backups-storage.md:209]: In cr azure it is repo3 by default
could we also use in key names repo3? 
My point people will use default settings (just uncomment rows in cr)

Comment (nastena1606) [reply docs/backups-storage.md:209]: Updated repo names according to cr.yaml

File: docs/assets/fragments/connectivity.txt [hunk 2]
```diff
@@ -1,36 +1,44 @@
...
     `<cluster_name>-pguser-<cluster_name>` (substitute `<cluster_name>` with
     the [name of your Percona Distribution for PostgreSQL Cluster](operator.md#metadata-name)).
-    The default variant will be `cluster1-pguser-cluster1`. You can use the
+    The default variant will be `cluster1-pguser-cluster1`. 
+
+2.  Use the
     following command to get the password of this user:
     
     ``` {.bash data-prompt="$" }
     $ kubectl get secret <cluster_name>-<user_name>-<cluster_name> -n postgres-operator --template='{{"{{"}}.data.password | base64decode{{"}}"}}{{"{{"}}"\n"{{"}}"}}'
```
[the remark this answers is not in the document]
Comment (nastena1606) [reply docs/assets/fragments/connectivity.txt:17]: Agree. yet we give the commands to basically copy&paste. I think it makes sense to leave values for quick install guides and replace namespace with placeholders for the rest of instructions.

**Review by egegunes — APPROVED** [review]

**Review by hors — APPROVED** [review]

File: docs/System-Requirements.md [hunk 3]
```diff
@@ -4,7 +4,13 @@ The Operator is validated for deployment on Kubernetes, GKE and EKS clusters.
 The Operator is cloud native and storage agnostic, working with a wide variety
 of storage classes, hostPath, and NFS.
 
-The Operator was developed and tested with PostgreSQL versions 12.14, 13.10, 14.7, and 15.2. Other options may also work but have not been tested. The Operator provides connection pooling based on [`pgBouncer`](https://www.pgbouncer.org/) 1.18.0 and high-availability implementation based on [`Patroni`](https://patroni.readthedocs.io/en/latest/) 3.0.1.
+## Supported versions
+
+The Operator {{ release }} is developed, tested and based on:
+
+* PostgreSQL 12.14, 13.10, 14.7, and 15.2 as the database. Other versions may also work but have not been tested.
```
Comment (fiowro) [inline docs/System-Requirements.md:11]:
```suggestion
* PostgreSQL 12.14, 13.10, 14.7, and 15.2 as the database (other versions may also work but have not been tested)
```

File: docs/assets/fragments/connectivity.txt [hunk 4]
```diff
@@ -1,36 +1,44 @@
...
     ```
 
-2. Run a container with `psql` tool and connect its console output to your
-    terminal. The following command will do this, naming the new Pod
+3. Create a pod and start Percona Distribution for PostgreSQL inside. The following command will do this, naming the new Pod
```
Comment (fiowro) [inline docs/assets/fragments/connectivity.txt:20]:
```suggestion
3. Create a Pod and start Percona Distribution for PostgreSQL inside. The following command will do this, naming the new Pod
```

File: docs/System-Requirements.md (hunk 3 above)
Comment (nastena1606) [reply docs/System-Requirements.md:11]: I got rid of the parentheses explicitly. According to the style guide one should avoid using them where possible (https://developers.google.com/style/parentheses)

Comment (fiowro) [reply docs/System-Requirements.md:11]: Well, internal list item ending with a period visually breaks the list. We have options not to end list items with a punctuation mark, or to put a comma in all paragraphs except the last one, and a period in the last one. If you put a period somewhere not at the end of the list, this conflicts with both schemes

File: docs/kubectl.md [hunk 5]
```diff
@@ -1,22 +1,33 @@
...
+A Kubernetes Operator is a special type of controller introduced to simplify complex deployments. The Operator extends the Kubernetes API with custom resources.
 
-## Pre-requisites
+The [Percona Operator for PostgreSQL](compare.md) is based on best practices for configuration and setup of a Percona Distribution for PostgreSQL cluster in a Kubernetes-based environment on-premises or in the cloud.
 
-The following tools are used in this guide and therefore should be preinstalled:
+We recommend installing the Operator with the [kubectl](https://kubernetes.io/docs/tasks/tools/) command line utility. It is the universal way to interact with Kubernetes. Alternatively, you can install it using the [Helm](https://github.com/helm/helm) package manager.
 
-1. The **Git** distributed version control system. You can install it following the [official installation instructions](https://github.com/git-guides/install-git).
+[Install with kubectl :material-arrow-down:](#pre-requisites){.md-button} [Install with Helm :material-arrow-right:](helm.md){.md-button}
```
Comment (nastena1606) [inline docs/kubectl.md:9]:
```suggestion
[Install with kubectl :material-arrow-down:](#prerequisites){.md-button} [Install with Helm :material-arrow-right:](helm.md){.md-button}
```

**Pull request merged**